### DISCUSSION TITLE
Is there any known reason why <CR> wouldn't be working with links?

### DISCUSSION THREAD

**Discussion by AndOrangutan** [nvim-neorg/neorg#142] [Q&A] [answered]
Hello,

Sorry for coming back so soon, but I am afraid that links are important for my note-taking workflow, and they currently aren't working for me. From what I understand from reading [this](https://github.com/nvim-neorg/neorg/pull/51), I should just be able to hit, and it will take me to that link. Instead of the proper behavior, it just takes me to the next line, like Nvim would in any other file. I looked at `:map` and couldn't see any conflicts, and I can't think of any plugins I have installed that would cause issues. I am up-to-date on the unstable branch. One thing, in that link from earlier, you mention moving to a special branch for the parser, but I couldn't find it on the repo, so I just assumed it was merged or something.

Any help is greatly appreciated :)

**Comment by vhyrro** [comment, marked as answer]
Hmm, it looks as if the keybinds were not applied at all. Are you enabling default keybinds in `core.keybinds`? Or are you rolling your own custom ones perhaps?

**Reply by AndOrangutan** [reply]
I believe I have it enabled properly:
``` lua
require('neorg').setup {
    -- Tell Neorg what modules to load
    load = {
        ["core.defaults"] = {}, -- Load all the default modules
        ["core.norg.concealer"] = {}, -- Allows for use of icons
        ["core.norg.dirman"] = { -- Manage your directories with Neorg
            config = {
                workspaces = {
                    my_workspace = "~/Dropbox/Neorg"
                },
                autodetect = true,
                autochdir = true,
            }
        },
        ["core.norg.completion"] = {
            config = {
                engine = "nvim-cmp"
            }
        },
        ["core.keybinds"] = {
            config = {
                default_keybinds = true,
                neorg_leader = "<Leader>o",
            },
        },
        ["core.integrations.telescope"] = {},
    },
}
```

**Reply by mrossinek** [reply]
I don't see anything out-of-ordinary here.. The only suggestion I have would be to try to enable logging as explained [here](https://github.com/nvim-neorg/neorg/wiki/Installation#configuring-the-logger) and watch for anything odd which may tell us what's going on here. With your above config I am unable to reproduce the issue.

You should try an isolated neovim instance with only neorg installed and see whether the issue still occurs. If not, it likely has to do with another part of your setup.

**Reply by AndOrangutan** [reply]
Where does the logger output to? Couldn't find where in where you linked.

**Reply by AndOrangutan** [reply]
After going through and disabling most plugins, it is working. I will report back with what plugin is the cause, otherwise I will count this as solved.

**Reply by AndOrangutan** [reply]
Weirdly enough, it was [gelguy/wilder.nvim](https://github.com/gelguy/wilder.nvim). Couldn't tell you why, but removing that plugin makes everything work :o. Thank you so much, you two.

**Reply by vhyrro** [reply]
Wow, i wonder what wilder.nvim is doing behind the scenes in that case? I can imagine them overwriting keybinds in command mode but in other modes? Weird. I might try looking into this at the end of the day today, see whether i can pinpoint the exact bit of code that's evoking the problem

**Reply by DevonMorris** [reply]
I'm seeing this same behavior.

Even running
```
:Neorg keybind norg core.norg.esupports.goto_link
```

Doesn't work
